### PR TITLE
Hotfix/tensorboard for cloud

### PR DESCRIPTION
**Pull request by HenriChabert**
Fix for [ch62284]

### CODE REVIEW

File: python-lib/dku_deeplearning_image/tensorboard_handler.py [hunk 1]
```diff
@@ -6,23 +6,36 @@
...
 from dku_deeplearning_image.dku_constants import TENSORBOARD_LOGS
 
 
-def get_logdir(folder_id):
+def load_logs_from_folder(folder_id):
     folder = dataiku.Folder(folder_id)
     try:
-        folder_path = folder.get_path()
-        return os.path.join(folder_path, TENSORBOARD_LOGS)
+        _ = folder.get_info()
```
Comment (andreybavt) [inline python-lib/dku_deeplearning_image/tensorboard_handler.py:17]: I guess you could combine this with the the next `folder.get_path()` call

File: python-lib/dku_deeplearning_image/tensorboard_handler.py [hunk 2]
```diff
@@ -35,34 +48,20 @@ def __get_custom_assets_zip_provider():
...
-    """
-    if issubclass(plugin_spec, base_plugin.TBLoader):
-        return plugin_spec()
-    if issubclass(plugin_spec, base_plugin.TBPlugin):
-        return base_plugin.BasicLoader(plugin_spec)
-    raise TypeError("Not a TBLoader or TBPlugin subclass: %r" % (plugin_spec,))
-
-
 def __get_tb_app(tensorboard_logs):
     plugins_or_loaders = tb_default.get_plugins()
```
Comment (andreybavt) [inline python-lib/dku_deeplearning_image/tensorboard_handler.py:58]: now that `make_plugin_loader` is not called and removed, this variable can be renamed to plugins, no?
btw, why did we remove `make_plugin_loader`?

**Review by andreybavt — CHANGES_REQUESTED** [review]

File: python-lib/dku_deeplearning_image/tensorboard_handler.py (hunk 2 above)
Comment (HenriChabert) [reply python-lib/dku_deeplearning_image/tensorboard_handler.py:58]: It seems that `standard_tensorboard_wsgi` accepts both plugins and loaders, so I do not need to transform the plugins into loaders. Do you remember why you wrote it in the first place in dip? Didn't `standard_tensorboard_wsgi` supported plugins?

File: python-lib/dku_deeplearning_image/tensorboard_handler.py (hunk 1 above)
Comment (andreybavt) [reply python-lib/dku_deeplearning_image/tensorboard_handler.py:16]: yep, sounds good

File: python-lib/dku_deeplearning_image/tensorboard_handler.py (hunk 2 above)
Comment (andreybavt) [reply python-lib/dku_deeplearning_image/tensorboard_handler.py]: nope, tbh don't remember why I had to do it. Well as long as it works and the name of the variable is not misleading anymore it's fine for me

**Review by andreybavt — APPROVED** [review]

**Pull request merged**